src/dart-2.md

```diff
@@ -137,11 +137,11 @@ environment:
 ```
```

Comment:
There is a repeated paragraph in lines 118 and 121 above:

> Once your package passes Dart 2 analysis, update the upper constraint to declare that the package is compatible with Dart 2:
>
> Once your package passes Dart 2 analysis, update the SDK constraint to declare that the package is compatible with Dart 2:

That needs to be fixed.

Reply:
Oops!